pkg/types/aws/platform.go

```diff
@@ -101,6 +101,12 @@ type Platform struct {
 	//
 	// +optional
 	LBType configv1.AWSLBType `json:"lbType,omitempty"`
+
+	// IgnoreS3Deletion is an optional field that can be used to make the S3 deletion optional
+	// during bootstrap destroy.
+	// See: https://issues.redhat.com/browse/CORS-2661
+	// +optional
+	IgnoreS3Deletion bool `json:"ignoreS3Deletion,omitempty"`
```

Comment:
"Ignore" seems a little negative and I think we should include "bootstrap" and/or "ignition" in the field name, so perhaps something like:

`PreserveBootstrapIgnition`
`SkipS3BootstrapDestroy`
`SkipBootstrapIgnitionDestroy`

I prefer the first one.